src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -198,22 +160,8 @@ void VM_Version::initialize() {
   // Create and print feature-string.
   char buf[(num_features+1) * 16]; // Max 16 chars per feature.
   jio_snprintf(buf, sizeof(buf),
-               "ppc64%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s",
-               (has_fsqrt()   ? " fsqrt"   : ""),
-               (has_isel()    ? " isel"    : ""),
+               "ppc64%s%s%s%s",
                (has_lxarxeh() ? " lxarxeh" : ""),
-               (has_cmpb()    ? " cmpb"    : ""),
-               (has_popcntb() ? " popcntb" : ""),
-               (has_popcntw() ? " popcntw" : ""),
-               (has_fcfids()  ? " fcfids"  : ""),
-               (has_vand()    ? " vand"    : ""),
-               (has_lqarx()   ? " lqarx"   : ""),
-               (has_vcipher() ? " aes"     : ""),
-               (has_vpmsumb() ? " vpmsumb" : ""),
-               (has_mfdscr()  ? " mfdscr"  : ""),
-               (has_vsx()     ? " vsx"     : ""),
-               (has_ldbrx()   ? " ldbrx"   : ""),
-               (has_stdbrx()  ? " stdbrx"  : ""),
                (has_vshasig() ? " sha"     : ""),
```

Comment:
The check for `has_vshasig()` is no longer needed (v2.07). I guess that " sha" should still get printed because some tests check for it.